examples/shadowmap/main.js

```diff
@@ -704,13 +626,46 @@
             this.updateShadowFormat();
             this.updateShadowMapSize();
 
-            this.updateDebugRTT();
+            var l, numLights;
+            numLights = ~~this._config.lightNum;
+            l = numLights;
+            while ( l-- ) {
```

Comment:
please use regular loop like in all example, I know it's faster but it's not really important here. Better to have consistent over performance for this kind of code